src/js/utils/PodUtil.js

```diff
@@ -0,0 +1,42 @@
+var PodUtil = {
+
+  /**
+   * Returns true if the container given matches
+   * the free-text given.
+   *
+   * @param {PodContainer} container - The Container to test
+   * @param {string} text - The text to test against
+   * @returns {boolean} - Returns true if the test passes
+   */
+  isContainerMatchingText(container, text) {
+    if (!text) {
+      return true;
+    }
+
+    return container.getName().indexOf(text) >= 0;
```

Comment:
@mlunoe I'm not sure I understand your comment. Whats the benefit of using `getId`?